ddtrace/span.py

```diff
@@ -49,13 +49,13 @@ def __init__(
         span_id=None,
         parent_id=None,
         start=None,
-        ctx=None,
+        context=None,
```

Comment:
Keeping `context` as optional kwarg. We will have time to transform it as a mandatory arg.